contracts/interchain-token-service/src/contract/execute/interceptors.rs

```diff
@@ -525,15 +542,23 @@ mod test {
         state::save_chain_config(
             &mut storage,
             &destination_chain,
-            &msg::ChainConfig {
-                chain: destination_chain.clone(),
-                its_edge_contract: "itsedgecontract".to_string().try_into().unwrap(),
-                truncation: TruncationConfig {
-                    max_uint_bits: 64.try_into().unwrap(),
-                    max_decimals_when_truncating: 6,
+            &ChainConfig::from_input(
+                msg::ChainConfig {
+                    chain: destination_chain.clone(),
+                    its_edge_contract: "itsedgecontract".to_string().try_into().unwrap(),
+                    truncation: TruncationConfig {
+                        max_uint_bits: 64.try_into().unwrap(),
+                        max_decimals_when_truncating: 6,
+                    },
+                    translation_contract: MockApi::default()
```

Comment:
use `address!()` since it creates object of `Address` type

Reply:
it will be validated later on, so still need to use the mock api to create

Reply:
created a new macro and used it here